sources/europe/fi/turku-orto-2018-true.geojson

```diff
@@ -0,0 +1,54 @@
+{
+    "type": "Feature",
+    "properties": {
+        "attribution": {
+            "required": true,
+            "text": "\u00a9 Turun kaupunki",
```

Comment:
you should be able to just insert the unicode © character

Reply:
Fixed. I just copied this from turku-orto.geojson so that file has this escape sequence as well